### PR TITLE
Tests/Toponaming: Add test for single shapes

### PR DESCRIPTION
**Pull request by CalligaroV**
Added test for Tag Info generated by makeShapeWithElementMap

**!!!IMPORTANT NOTE!!!:** This test makes sense only for single shapes and shapes that aren't generated by geometrical operations.

A more robust version of this test will be developed, if needed, to test more complex shapes

### CODE REVIEW

**Comment by chennes** [comment]
@bgbsww this look good to you?

File: tests/src/Mod/Part/App/TopoShapeMakeShapeWithElementMap.cpp [hunk 1]
```diff
@@ -340,3 +340,45 @@ TEST_F(TopoShapeMakeShapeWithElementMapTests, findMakerOpInElementMap)
...
+                         // more or less the same of nonMappableSources
+
+    // Act and assert
+    // Testing with all the source TopoShapes
+    for (const auto& source : sources) {
+        Part::TopoShape tmpShape {source.getShape()};
+        tmpShape.makeShapeWithElementMap(source.getShape(), *Mapper(), sources);
+
+        // For all the mappedElements ...
+        for (const auto& mappedElement : tmpShape.getElementMap()) {
```
Comment (bgbsww) [inline tests/src/Mod/Part/App/TopoShapeMakeShapeWithElementMap.cpp:371]: I think this test looks good.  Note that if getElementMap() returns empty, then no testing will be done at all.  getElementMap() is tested separately, so not necessarily required here.  I think I'd add an expect for the length of the element map just as a compulsive double check that the test is doing something.

**Review by bgbsww — APPROVED** [review]
Change not required, just a thought.

File: tests/src/Mod/Part/App/TopoShapeMakeShapeWithElementMap.cpp (hunk 1 above)
Comment (CalligaroV) [reply tests/src/Mod/Part/App/TopoShapeMakeShapeWithElementMap.cpp:371]: Thanks for the hint!

I'll add a check right now.

**Pull request merged**